### PR TITLE
Add Open AI service

### PR DESCRIPTION
**Pull request by ReebalSami**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/backend/model/openai/ChatGptChoice.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatGptChoice {
+    /**
+     {
+     "message": {
+     "content": "This is a test!"
+     }
+     }
+     */
```
Comment (MarcellNF) [inline backend/src/main/java/de/neuefische/backend/model/openai/ChatGptChoice.java]: you dont want comments in your production code

File: backend/src/main/java/de/neuefische/backend/model/openai/ChatGptMessage.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+     {
+     "messages":
+     [
+     {
+     "role": "user",
+     "content": "Say this is a test!"
+     }
+     ]
+     }
+     */
```
Comment (MarcellNF) [inline backend/src/main/java/de/neuefische/backend/model/openai/ChatGptMessage.java]: you dont want comments in your production code

File: backend/src/main/java/de/neuefische/backend/model/openai/ChatGptRequest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+public class ChatGptRequest {
+    /**
+     {
+     "model": "gpt-3.5-turbo",
+     "response_format": { "type": "json_object" },
+     "messages": [{"role": "user", "content": "Say this is a test!"}],
+
+     }
+     */
```
Comment (MarcellNF) [inline backend/src/main/java/de/neuefische/backend/model/openai/ChatGptRequest.java]: you dont want comments in your production code

File: backend/src/main/java/de/neuefische/backend/model/openai/ChatGptResponse.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+     [
+     {
+     "message": {
+     "role": "assistant",
+     "content": "This is a test!"
+     }
+     }
+     ]
+     }
+     */
```
Comment (MarcellNF) [inline backend/src/main/java/de/neuefische/backend/model/openai/ChatGptResponse.java]: you dont want comments in your production code

File: frontend/src/components/layout/Navigation.css [hunk 5]
```diff
@@ -1,3 +1,5 @@
+/* Navigation.css */
```
Comment (MarcellNF) [inline frontend/src/components/layout/Navigation.css]: you dont want comments in your production code

File: frontend/src/components/layout/Header.css [hunk 6]
```diff
@@ -1,8 +1,12 @@
+/* Header.css */
```
Comment (MarcellNF) [inline frontend/src/components/layout/Header.css]: you dont want comments in your production code

**Review by MarcellNF — CHANGES_REQUESTED** [review]
so far so good, but you have many comments in your code and you dont want it in production :-)

File: backend/src/main/java/de/neuefische/backend/model/openai/ChatGptChoice.java (hunk 1 above)
Comment (ReebalSami) [reply backend/src/main/java/de/neuefische/backend/model/openai/ChatGptChoice.java]: done!

File: backend/src/main/java/de/neuefische/backend/model/openai/ChatGptMessage.java (hunk 2 above)
Comment (ReebalSami) [reply backend/src/main/java/de/neuefische/backend/model/openai/ChatGptMessage.java]: done!

File: backend/src/main/java/de/neuefische/backend/model/openai/ChatGptRequest.java (hunk 3 above)
Comment (ReebalSami) [reply backend/src/main/java/de/neuefische/backend/model/openai/ChatGptRequest.java]: done!

File: backend/src/main/java/de/neuefische/backend/model/openai/ChatGptResponse.java (hunk 4 above)
Comment (ReebalSami) [reply backend/src/main/java/de/neuefische/backend/model/openai/ChatGptResponse.java]: done!

File: frontend/src/components/layout/Header.css (hunk 6 above)
Comment (ReebalSami) [reply frontend/src/components/layout/Header.css]: done!

File: frontend/src/components/layout/Navigation.css (hunk 5 above)
Comment (ReebalSami) [reply frontend/src/components/layout/Navigation.css]: done!

**Review by MarcellNF — APPROVED** [review]
LGTM!:)

**Pull request merged**